components/onboarding-modal/OnboardingProfileCard.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Flex, Box } from '@rebass/grid';
+import styled from 'styled-components';
+
+import Avatar from '../../components/Avatar';
+import StyledTag from '../../components/StyledTag';
+
+const Admin = styled(StyledTag)`
+  font-size: 14px;
+  border-top-right-radius: 50px;
+  border-bottom-right-radius: 50px;
+`;
+
+class OnboardingProfileCard extends React.Component {
+  static propTypes = {
+    user: PropTypes.object,
+  };
+
+  render() {
+    const { user } = this.props;
+    const { imageUrl, name, type } = user;
+
+    return (
+      <Flex my={1} ml={2}>
+        <Admin textTransform="none">
+          <Flex alignItems="center">
+            <Avatar src={imageUrl ? imageUrl : null} radius={15} name={name} type={type} />
```

Comment:
As seen above, what you actually pass with `user` is in fact a collective. So once the prop are renamed, you can just write:

```suggestion
            <Avatar radius={15} collective={collective} />
```